src/mysql.c

```diff
@@ -995,6 +1024,54 @@ static int mysql_read (user_data_t *ud)
 		{
 			counter_submit ("mysql_slow_queries", NULL , val, db);
 		}
+ 		else if (strncmp (key, "Queries", strlen ("Queries")) == 0) 
+ 		{
+ 			counter_submit ("mysql_queries", NULL , val, db);
```

Comment:
The type *mysql_queries* does not exist. I propose to use the *total_operations* type and set the *type_instance* to "query".